performance-test/src/gatling/java/org/opensearch/dataprepper/test/performance/SlowBurnTraceSimulation.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.test.performance;
+
+import io.gatling.javaapi.core.CoreDsl;
+import io.gatling.javaapi.core.ScenarioBuilder;
+import io.gatling.javaapi.core.Simulation;
+import org.opensearch.dataprepper.test.performance.tools.Protocol;
+import org.opensearch.dataprepper.test.performance.tools.TraceChain;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+
+public class SlowBurnTraceSimulation extends Simulation {
```

Comment:
The naming choice is a little confusing. `FixedClientTraceSimulation` and `SlowBurnTraceSimulation` use `Slow Burn` scenario. Yet the difference is fixed and ramping clients. I might recommend `FixedClientTraceSimulation` and `RampUpClientTraceSimulation`